automated-comments/go/hamming/lossy_rune_to_byte_conversion.md

```diff
@@ -0,0 +1,5 @@
+You are converting a `rune` to a `byte`. Go `runes` can consist of multiple `bytes`, so this conversion can cause you problems if you have to deal with non-ASCII characters. However, in this case it's OK, because the input string contains only ASCII (single-byte) characters.
+
+To get a deeper understanding of these issues you might like to read [Strings, bytes, runes and characters in Go](https://blog.golang.org/strings) on the Go blog.
+    
+You can also add more tests to  this exercise containing non-ASCII characters and see what happens.
```

Comment:
Maybe:

> The `rune` to `byte` conversion is safe here, because the input string contains only ASCII characters. Go `runes` can consist of multiple `bytes`, though, which can cause trouble when dealing with non-ASCII characters.
> 
> For more on this, read [Strings, bytes, runes and characters in Go](https://blog.golang.org/strings) on the Go blog.
>
> To see this in practice, add a test containing non-ASCII characters.